### PR TITLE
Inter font Updated

### LINKED ISSUE #39: Applying Inter Font Everywhere
To apply the inter font family in the complete project.
Better to add it in index.css or globalstyle.css so that all the project can have it.

Also include the bold version of the font along side it.

### PR DESCRIPTION
**Pull request by Shruti025**
Fixes: #39 

I have updated the website to have Inter Font everywhere.

### CODE REVIEW

**Comment by Shruti025** [comment]
@anushbhatia kindly review this PR

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
```
Comment (anushbhatia) [inline src/index.css]: import the complete family major weights like:
Regular
Medium
Semi Bold
Bold
Extra Bold

Comment (Shruti025) [reply src/index.css]: Done

**Review by smaranjitghose — APPROVED** [review]

**Comment by Shruti025** [comment]
Hey @smaranjitghose can you please add the JWOC and difficulty labels else the PR wont be considered

**Pull request merged**